### PR TITLE
Update copy about IRS CTC Update portal

### PR DESCRIPTION
**Pull request by embarnard**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2731,16 +2731,18 @@ en:
...
-            body3: "You might choose to opt out of receiving monthly Child Tax Credit payments if:"
+            body2_html: "If your family situation has changed you will soon be able to use the <strong><a rel='noopener' target='_blank' href='https://www.irs.gov/credits-deductions/child-tax-credit-non-filer-sign-up-tool'>IRS CTC Update Portal</a></strong> to:"
+            body3_html: "You can <i>currently</i> use the Update Portal to:"
+            body4: "You might choose to opt out of receiving monthly Child Tax Credit payments if:"
             update_items:
-              - "Update the number of eligible dependents in your household (for example if a child has moved out)"
+              - "Update the number of eligible dependents in your household (for example if a child has moved out, or a new baby was born)"
               - "Report significant changes in household income"
-              - "Opt out of receiving monthly payments, and choose to receive the payment next year when you submit your 2021 tax return"
+            current_use_item: "Opt out of receiving monthly payments, and choose to receive the payment next year when you submit your 2021 tax return"
```
Comment (jenny-heath) [inline config/locales/en.yml:2740]: consider naming this key something more intuitive - maybe `payment_next_year` or `opt_out_monthly_payments`

**Review by jenny-heath — APPROVED** [review]
Next time please include a screenshot in the PR when there are visual changes

**Comment by embarnard** [comment]
> Next time please include a screenshot in the PR when there are visual changes

@jenny-heath  A screenshot of what it was before and what it is now?

**Comment by embarnard** [comment]
> Next time please include a screenshot in the PR when there are visual changes

Screen Shot 2021-06-28 at 6 18 36 PM

**Pull request merged**